.github/workflows/ci.yml

```diff
@@ -19,36 +19,38 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       matrix:
-        nxf_ver: ['19.10.0', '']
+        nxf_ver: ['20.04.0', '']
```

Comment:
```suggestion
        nxf_ver: ['20.10.0', '']
```

Reply:
I recall there were some issues regarding a variable name with `20.04.0` so I'd recommend going straight to `20.10.0`